book/src/developers/contributing/build_instructions/linux.md

```diff
@@ -132,7 +132,7 @@ Open the file:
 ```sh
 $ sudo nano /etc/systemd/system/trin.service.d/override.conf
 ```
-Paste the following, replace the Infura ID with your own.
```

Comment:
Side note: I noticed this line but it doesn't fit the rest of the text, so I assumed it's leftover from some previous versions.